docs/main/create-ocp-project/README.md

```diff
@@ -14,97 +14,176 @@ The following table lists the configurable parameters of the Create OCP project
+| `NOTIFICATIONS_BEARER_TOKEN`      | The authorization bearer token to use to send notifications. This can be found in the Backstage configuration under `app.backend.auth.externalAccess` ([example](https://github.com/parodos-dev/orchestrator-helm-operator/blob/main/docs/release-1.2/existing-rhdh.md#app-config-configmap)) | ✅ | |
+| `JIRA_URL`      | The Jira server URL, e.g `https://your-instance.atlassian.net` | ✅ | |
+| `JIRA_USERNAME`      | The Jira username, e.g `you@company.com` | ✅ | |
+| `JIRA_API_TOKEN`      | The Jira API token associated with the username. If you're using Jira cloud, you can obtain a token [at this link](https://id.atlassian.com/manage-profile/security/api-tokens). | ✅ | |
+| `OCP_API_SERVER_URL`      | The OpenShift API server url, e.g `https://api.cluster.hostname.com:6443` | ✅ | |
+| `OCP_API_SERVER_TOKEN`      | The authorization bearer token to use when sending request to OpenShift | ✅ | |
 
-To do so, edit the secret `${WORKFLOW_NAME}-creds` and set those values and the one of `NOTIFICATIONS_BEARER_TOKEN`:
-```
-WORKFLOW_NAME=create-ocp-project
-oc -n sonataflow-infra patch secret "${WORKFLOW_NAME}-creds" --type merge -p '{
-   "data":{
-      "NOTIFICATIONS_BEARER_TOKEN":"'$(oc get secrets -n rhdh-operator backstage-backend-auth-secret -o go-template='{{ .data.BACKEND_SECRET  }}')'"
-   },
-   "stringData":{
-      "JIRA_API_TOKEN":"{{ JIRA_API_TOKEN }}",
-      "OCP_API_SERVER_TOKEN":"{{ OCP_API_SERVER_TOKEN }}"
-   }
-}'
+To obtain an OpenShift API token, create a Service Account, assign permissions to it, and request a token:
+
+```bash
+oc create sa orchestrator-ocp-api
+oc adm policy add-cluster-role-to-user cluster-admin -z orchestrator-ocp-api
+oc create token orchestrator-ocp-api
 ```
-If you are using Jira cloud, you can generate the `JIRA_API_TOKEN` using https://id.atlassian.com/manage-profile/security/api-tokens 
 
-The `OCP_API_SERVER_TOKEN` should be associated with a service account.
+### Add the Environment Variables to a Secret
+
+The Helm Chart installation creates a `create-ocp-project-creds` Secret in the
+namespace where it's installed. You'll update this Secret with your environment
+variable values.
+
+> [!NOTE]
+> Updating the Secret does not automatically restart the workflow Pod, nor update the associated Sonataflow CR. If you update the keys in the secret you must update the Sonataflow CR - as shown below - to reference them. If you update the values in the Secret you must delete the existing `create-ocp-project` workflow Pod to create a new Pod that uses the new values.
+
+> [!WARNING]
+> If you run the `helm upgrade` command, the values of the Secret are reset.
+
+Run the following command to do so. Replace the example values with the
+correct values for your environment:
+
+```bash
+export TARGET_NS='sonataflow-infra'
+export WORKFLOW_NAME='create-ocp-project'
+
+export NOTIFICATIONS_BEARER_TOKEN='token_from_backstage_appconfig'
```

Comment:
While we are at it, let's get the token directly:
```
export NOTIFICATIONS_BEARER_TOKEN=$(oc get secrets -n rhdh-operator backstage-backend-auth-secret -o go-template='{{ .data.BACKEND_SECRET  }}')
```

Reply:
@gabriel-farache I added comments in another thread here to discuss this.